xml/security-barbican.xml

```diff
@@ -484,8 +574,8 @@ ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
       </entry>
       <entry>
        <para>
-        Barbican specific role with privilege to create/modify/list keys,
-        certificates.
+        Barbican specific role with privilege to create/modify/list/delete
```

Comment:
Usually, we try to avoid slashes, so _create, modify, list, and delete_ is preferable.